### PR TITLE
[bubble] Fixing ad-hoc metric labels

### PR DESCRIPTION
**Pull request by john-bodley**
This PR fixes the bubble chart labels which previously rendered as `[object Object]` if the metric was an ad-hoc metric.

to: @GabeLoins @graceguo-supercat @michellethomas @mistercrunch @timifasubaa

### CODE REVIEW

**Review by conglei — COMMENTED** [review]
What about non ad-hoc metric field?  seems like we should first test if `fd.x` is an `object` or it is just a `string`

File: superset/assets/src/visualizations/nvd3_vis.js [hunk 1]
```diff
@@ -329,9 +329,9 @@ export default function nvd3Vis(slice, payload) {
             `<tr><td style="color: ${p.color};">` +
               `<strong>${p[fd.entity]}</strong> (${p.group})` +
             '</td></tr>');
-          s += row(fd.x, formatter(p.x));
-          s += row(fd.y, formatter(p.y));
-          s += row(fd.size, formatter(p.size));
+          s += row(fd.x.label, formatter(p.x));
```
Comment (conglei) [inline superset/assets/src/visualizations/nvd3_vis.js]: What about non ad-hoc metric field? seems like we should first test if fd.x is an object or it is just a string. Same thing for the other two.

Comment (john-bodley) [reply superset/assets/src/visualizations/nvd3_vis.js]: @conglei only ad-hoc metrics are defined for bubble-charts.

**Comment by john-bodley** [comment]
@conglei thanks for bringing this to my attention. I've updated the logic.

**Review by graceguo-supercat — APPROVED** [review]
LGTM

**Pull request merged**